plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -992,15 +998,19 @@ private boolean isMaterializedView(Table table)
         }
 
         Table materializedView = tableOptional.get();
+        Optional<String> storageTable = Optional.ofNullable(materializedView.getParameters().get(STORAGE_TABLE));
+        if (storageTable.isEmpty()) {
```

Comment:
I don't think it should ever by empty, `createMaterializedView` always adds storage table.
`checkState` instead?